MicroServices/ShopFlowPro-service/src/main/java/com/wsu/shopflowproservice/service/ServiceService.java

```diff
@@ -0,0 +1,39 @@
+package com.wsu.workorderproservice.service;
+
+import com.wsu.workorderproservice.repository.ServiceRepository;
+import com.wsu.workorderproservice.dto.ServiceDTO;
+import com.wsu.workorderproservice.model.Service;
+import com.wsu.workorderproservice.exception.DatabaseErrorException;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.cglib.core.CollectionUtils;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@RequiredArgsConstructor
```

Comment:
Missing Service annotation to declare this class as service for Spring Context bean class.